src/services/leafletWatchHelpers.js

```diff
@@ -20,7 +20,17 @@ angular.module("leaflet-directive")
   to set watching to once and its watch depth.
   */
   var _maybeWatch = function(scope, thingToWatchStr, watchOptions, initCb){
-      return _maybe(scope, '$watch', thingToWatchStr, watchOptions, initCb);
+      var localWatchOptions, watchMethod;
```

Comment:
Well if it is `normal`; that should be default / what it is when it is when it is `undefined`. No would mean off. Hmm not sure; one thing it needs to be backwards compatible as well as I heavily use this feature. I pretty much disable watching all together when I use this lib.